### PR TITLE
Make actionview an optional dependency

### LINKED ISSUE #5: Errors when running on Ruby 2.3 and Rails 2.3 LTS
When running `bundle_report outdated` on a Rails 2.3 LTS project the following errorr is returned:

`You have already activated activesupport 5.2.4.4, but your Gemfile requires activesupport 2.3.18.31`

The complete output is:

```
WARN: Unresolved specs during Gem::Specification.reset:
      i18n (< 2, >= 0.7)
      minitest (~> 5.1)
WARN: Clearing out unresolved specs.
Please report a bug if this causes problems.
/usr/local/bundle/gems/next_rails-1.0.2/exe/bundle_report:8:in `require': cannot load such file -- next_rails (LoadError)
	from /usr/local/bundle/gems/next_rails-1.0.2/exe/bundle_report:8:in `block in <top (required)>'
/usr/local/bundle/gems/bundler-1.12.5/lib/bundler/runtime.rb:35:in `block in setup': You have already activated activesupport 5.2.4.4, but your Gemfile requires activesupport 2.3.18.31. Prepending `bundle exec` to your command may solve this. (Gem::LoadError)
	from /usr/local/bundle/gems/bundler-1.12.5/lib/bundler/runtime.rb:20:in `map'
	from /usr/local/bundle/gems/bundler-1.12.5/lib/bundler/runtime.rb:20:in `setup'
	from /usr/local/bundle/gems/bundler-1.12.5/lib/bundler.rb:95:in `setup'
	from /usr/local/bundle/gems/bundler-1.12.5/lib/bundler/setup.rb:9:in `<top (required)>'
	from /usr/local/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:133:in `require'
	from /usr/local/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:133:in `rescue in require'
	from /usr/local/lib/ruby/site_ruby/2.3.0/rubygems/core_ext/kernel_require.rb:40:in `require'
	from /usr/local/bundle/gems/next_rails-1.0.2/exe/bundle_report:57:in `<top (required)>'
	from /usr/local/bundle/bin/bundle_report:22:in `load'
	from /usr/local/bundle/bin/bundle_report:22:in `<main>'
```

I think if next_rails avoided `actionview` as a strict dependency by removing `spec.add_runtime_dependency "actionview"` from the gemspec it might help. Actionview is in fact optional from what I've seen in [the code](https://github.com/fastruby/next_rails/blob/master/lib/next_rails/gem_info.rb#L1) so I believe this wouldn't be an issue.

There is also https://github.com/fastruby/next_rails/issues/4 which is somewhat related.

### PR DESCRIPTION
**Pull request by manuca**
Resolves #5

### CODE REVIEW

File: next_rails.gemspec [hunk 1]
```diff
@@ -24,10 +24,11 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "colorize", ">= 0.8.1"
+  spec.add_dependency "activesupport", "< 6.0"
```
Comment (etagwerker) [inline next_rails.gemspec:27]: @manuca How come this is necessary now?

**Review by etagwerker — COMMENTED** [review]
@manuca I understand that actionview can be an optional dependency because of this: 

https://github.com/fastruby/next_rails/blob/master/lib/next_rails/gem_info.rb#L1-L5

And this: https://github.com/fastruby/next_rails/blob/master/lib/next_rails/gem_info.rb#L55-L59

But I have a question about activesupport.

File: next_rails.gemspec (hunk 1 above)
Comment (manuca) [reply next_rails.gemspec:27]: > @manuca How come this is necessary now?

@etagwerker This is the culprit:
https://github.com/manuca/next_rails/blob/make-actionview-optional/exe/bundle_report#L59

Comment (etagwerker) [reply next_rails.gemspec:27]: @manuca Weird. And where is that code getting called? What methods are we using from active_support?

Comment (manuca) [reply next_rails.gemspec:27]: Which to be honest I have no idea why we're depending on it as there are no `acts_like?` calls anywhere as far as I've searched.

For reference this is `acts_like.rb`:
https://github.com/rails/rails/blob/5-2-stable/activesupport/lib/active_support/core_ext/object/acts_like.rb

Comment (manuca) [reply next_rails.gemspec]: @etagwerker I removed the `activesupport` dependency. Specs pass and `bundle_report` seems to work as expected.

**Review by etagwerker — APPROVED** [review]
@manuca Great, thanks! 👍

File: next_rails.gemspec (hunk 1 above)
Comment (etagwerker) [reply next_rails.gemspec]: @manuca Interesting. I ran into the reference to `acts_like?` in v1.0.3: #21. I guess it is getting used in the exe command.

**Pull request merged**